test/README.md

```diff
@@ -21,14 +21,14 @@ cd serve
 This would build a docker Image with a tag torchserve:1.0 in which we would run our Regression Tests.
```

Comment:
@shivamshriwas Please update in the instructions as well (:latest tag)

Reply:
Done